README.md

```diff
@@ -64,7 +65,10 @@ None.
 
 ## Dependencies ##
 
-None.
+This Ansible role depends on
+[cisagov/ansible-role-backports](https://github.com/cisagov/ansible-role-backports),
+since on Debian Buster we need a newer version of the `golang` package
+in order to build Go-based projects.
```

Comment:
Just wanted this more in-line with how we list dependencies in other Ansible roles. We don't usually explain the dependencies so also feel free to reduce it just the list item.

```suggestion
- [cisagov/ansible-role-backports](https://github.com/cisagov/ansible-role-backports):
  On Debian Buster we need a newer version of the `golang` package in order to
  build Go-based projects.
```